<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	Background: #002200;
	color: #FFC;
	border: &quot;black&quot;;
	border-width: 1px;
	border-style: solid;
    border-radius: 4px;
	font-family: &quot;Baskerville&quot;
}

QLabel
{
	Background: #003300
}

QPushButton
{
	Background: #300;
	border: &quot;black&quot;;
	border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}

QPushButton:Disabled
{
	Background: #330
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="playButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>32</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="moneyLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="dealerCardLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cards:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="standButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>32</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Stand</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="playerCounterLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="dealerTitleLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">BackGround: #300</string>
      </property>
      <property name="text">
       <string>Dealer's Hand</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="playerCardLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cards:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="dealerCounterLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="betLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your Bet: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="playerTitleLabel">
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">BackGround: #300</string>
      </property>
      <property name="text">
       <string>Your Hand</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="betButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>32</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Update Bet</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="hitButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>32</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Hit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
